apps/web/utils/links.ts

```diff
@@ -0,0 +1,7 @@
+export const discord = 'https://discord.gg/qJWcsS9TyT';
```

Comment:
because /chat redirects to the discord invite
```suggestion
export const discord = '/chat';
```